### PR TITLE
Fix logo scraper issues

### PR DESCRIPTION
**Pull request by tspenov**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex [hunk 1]
```diff
@@ -42,33 +38,42 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.LogoFetcher do
...
+    {:ok, remote_projects} = CryptocurrencyInfo.fetch_data([project])
+    logo_map = remote_projects |> Enum.into(%{}, fn ci -> {ci.slug, ci.logo} end)
+    update_project_logos(project, logo_map, dir_path)
 
-      _ ->
+    Temp.cleanup()
+  end
+
+  defp update_project_logos(project, logo_map, dir_path) do
+    coinmarketcap_id = project.coinmarketcap_id
```
Comment (IvanIvanoff) [inline lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex:57]: Use `Project.coinmarketcap_id` function to get the `coinmarketcap_id`. This field is deprecated and not populated properly.
You can also preload the proper association for that.

Comment (tspenov) [reply lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex]: fixed

File: lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex [hunk 2]
```diff
@@ -42,33 +38,41 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.LogoFetcher do
...
+           update_local_project(project, %{
+             logo_url: uploaded_filepath
+           }) do
+      Logger.info("#{@log_tag} Successfully updated logos for project: #{project.slug}")
+    else
+      {:logo_has_changed?, false} ->
         :ok
+
+      error ->
+        error
```
Comment (IvanIvanoff) [inline lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex:75]: This might be `nil` because of `Project.coinmarketcap_id`. Will this be a problem?

**Review by IvanIvanoff — APPROVED** [review]

File: lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex (hunk 2 above)
Comment (tspenov) [reply lib/sanbase/external_services/coinmarketcap/logo_fetcher.ex:75]: the result is not used, I can remove the whole else

**Pull request merged**